keyboards/ergodash/rev1/rev1.c

```diff
@@ -30,6 +30,7 @@ void matrix_init_kb(void) {
 	matrix_init_user();
 };
 
+__attribute__ ((weak))
```

Comment:
Actually, this function should be removed.

All it's doing is the "shutdown sound", and it's actually overriding the globally configured one.

Reply:
It looks like the globally defined one is empty?

https://github.com/qmk/qmk_firmware/blob/c8db24ce345e5a87073238ff150336bf6033b9d0/quantum/quantum.c#L1522

Or am I looking in the wrong place?

Reply:
This is what this function is doing in this file:
https://github.com/qmk/qmk_firmware/blob/c8db24ce345e5a87073238ff150336bf6033b9d0/keyboards/ergodash/rev1/rev1.c#L33-L40

This is the function that actually calls that:
https://github.com/qmk/qmk_firmware/blob/c8db24ce345e5a87073238ff150336bf6033b9d0/quantum/quantum.c#L163-L187

Basically, this function is unneeded here, since it's legacy code. If you check some of the older planck stuff, it did this, until jack moved it into the core code.